pom.xml

```diff
@@ -85,7 +85,7 @@ flexible messaging model and an intuitive client API.</description>
     <pulsar.client.python.version>3.2.0</pulsar.client.python.version>
 
     <!--config keys to configure test selection -->
-    <include>**/Test*.java,**/*Test.java,**/*Tests.java,**/*TestCase.java</include>
+    <include>*</include>
```

Comment:
Let's make this more restrictive since it was the default for Surefire, see https://maven.apache.org/surefire/maven-surefire-plugin/examples/inclusion-exclusion.html#inclusions .

Making it * results in scanning all class files, also class files for lambdas and inner classes.
That was the reason why I changed it a long time ago.

Reply:
Thank you for the explanation. After reading your description, I now agree with the current setting. I'll only modify the existing class name in this PR, such as `**/*Test2.java`.

By the way, I already noticed two patterns before publishing this PR.

* `Test` suffix is missing in a class name
  - They can't be filtered by class name.
* extra suffix is added
  - e.g. `**/*Test2.java`
  - They may be filtered by class name.

Reply:
I found omitted tests by following commands.
```
$ find . ! -name '*Test*' -name "*.java" -exec grep -H "@Test" {} \;
$ find . -regextype posix-basic -regex '.*/[^/]\+Test[^/]\+\.java$' ! -regex '.*Test\(s\)\?.java$' -exec grep -H "@Test" {} \;
```

exceptions:
* abstract classes
* org.apache.pulsar.metadata.MetadataBenchmark
  - It doesn't have to run in unit tests.
* backward compatibility tests
  - e.g. `SmokeTest2_5.java`